src/FwdState.cc

```diff
@@ -725,16 +725,13 @@ FwdState::connectDone(const Comm::ConnectionPointer &conn, Comm::Flag status, in
         return;
     }
 
-    serverConn = conn;
-    debugs(17, 3, HERE << serverConnection() << ": '" << entry->url() << "'" );
-
-    closeHandler = comm_add_close_handler(serverConnection()->fd, fwdServerClosedWrapper, this);
+    debugs(17, 3, HERE << conn << ": '" << entry->url() << "'" );
```

Comment:
`HERE`

Extra space at the end of parameters.

I would also remove manual quotes around url() but that is your call.

```suggestion
    debugs(17, 3, conn << " for " << entry->url());
```

Reply:
commit 6083bf8354